qiskit/qasm/libs/qelib1.inc

```diff
@@ -260,7 +260,5 @@ gate c4x a,b,c,d,e
 {
     h e; cu1(pi/2) d,e; h e;
     c3x a,b,c,d;
-    h e; cu1(-pi/2) d,e; h e;
-    c3x a,b,c,d;
-    c3sqrtx a,b,c,e;
+    c3sx a,b,c,e;
```

Comment:
Is this necessary?  I'm not keen on changing definitions unless it's actually wrong.